### ISSUE TITLE
Please add support for CBZ output

### ISSUE THREAD

**Issue by Poyeyo** [leotaku/kojirou#16]
I have a Pocketbook and a Kobo reader.

They are more compatible with CBZ than AWS3 output.

**Comment by leotaku** [comment]
There already seem to be other projects that support saving manga from MangaDex to CBZ, why not use them? 

Alternatively, as CBZ from my understanding just seems to be a sorted list of images collected in a ZIP archive, so you might also just want to use a more fully-featured downloader and collect them into archives yourself. Using strategy would also help you ever need to get your manga from somewhere else than MangaDex.

In short, I currently don't really see much value in adding CBZ support to kojirou. Also, I am not really familiar with the Comic Book Archive format, do you know if it has a standardized way of adding metadata or a chapter index to archives?

**Comment by Poyeyo** [comment]
The autocrop feature is what makes me want to keep using this tool, instead of others.

**Comment by leotaku** [comment]
In that case, I would recommend you use a downloader that outputs image files, use ImageMagick's [trim](http://www.imagemagick.org/script/command-line-options.php#trim) functionality to automatically crop your images, and then combine them into a ZIP file.

As I've said, I don't see myself adding CBZ support in the very near future, but with the announcement that Kindle readers do/will support EPUB, there is now a better chance that I will add support for other formats at some point. I assume that you will not want to wait for this, though, which is why I would recommend the above option.

**Comment by celogeek** [comment]
Also you can use this tools to create the AWZ3, then use calibre to create the CBZ ( or zip ).
